spec/factories/disk.rb

```diff
@@ -1,4 +1,11 @@
 FactoryGirl.define do
   factory :disk do
+    trait(:hdd) do
+      device_name "test_disk"
```

Comment:
I'm wondering if we need this attribute at all....is it essential to defining this type of disk or is it serving as an illustration only? Same question for all the attributes specified....I would expect that it is only `device_type` that is definitive.